director/templates/index.html

```diff
@@ -273,6 +275,7 @@
       const API_URL="%% config.API_URL %%";
       const FLOWER_URL="%% config.FLOWER_URL %%";
       const REFRESH_INTERVAL=%% config.REFRESH_INTERVAL | int %%;
+      const REPO_LINK="%% config.REPO_LINK %%";
```

Comment:
Same comment as `self.REPO_LINK` in the `settings.py` file.

Reply:
see  answer to https://github.com/ovh/celery-director/pull/141/files/3d2c0bfcac90c9740b84d34fb3514fd3f175951b#r849687363